rules/windows/defense_evasion_amsienable_key_mod.toml

```
@@ -29,16 +29,20 @@ This rule monitors the modifications to the Software\\Microsoft\\Windows Script\
 
 #### Possible investigation steps
 
-- Identify the user that performed the action.
-- Check whether this user should be doing this kind of activity.
-- Investigate program execution chain (parent process tree).
-- Investigate other alerts related to the user/host in the last 48 hours.
+- Identify the user account that performed the action and whether it should perform this kind of action.
+- Investigate the process execution chain (parent process tree) for unknown processes. Examine their executable files
+for prevalence, whether they are located in expected locations, and if they are signed with valid digital signatures.
+- Investigate other alerts associated with the user/host during the past 48 hours.
 - Investigate the execution of scripts and macros after the registry modification.
-- Retrieve script/office files:
-  - Use a sandboxed malware analysis system to perform analysis.
-    - Observe attempts to contact external domains and addresses.
-  - Use the PowerShell Get-FileHash cmdlet to get the SHA-256 hash value of the file.
-    - Search for the existence of this file in resources like VirusTotal, Hybrid-Analysis, CISCO Talos, Any.run, etc.
+- Retrieve script/office files and determine if they are malicious:
```

Is this referring to Microsoft Office? Prob better capitalize then:
```suggestion
- Retrieve scripts or Microsoft Office files and determine if they are malicious:
```